src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,249 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private Node<K, V>[] arrayNodes;
+    private final int defaultCapacity = 16;
+    private final float loadFactor = 0.75f;
```

Comment:
lets make this variables as constants https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names